### PR TITLE
Minor syntax fix for unexpected `}`

### PR DESCRIPTION
**Pull request by Tkael**

### CODE REVIEW

File: SpeechResponder/Cottle.xshd [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <Color name="Body text" foreground="DarkSlateGray" exampleText="This is body text"/>
   <Color name="Comment" foreground="Green" fontStyle="italic" exampleText="{_ This is a comment. }"/>
   <Color name="Code" foreground="Purple" exampleText="{ variableName }"/>
-  <Color name="Delimiter" foreground="Purple" fontWeight="bold" exampleText="{ | }"/>
+  <Color name="Delimiter" foreground="Purple"  background="LightGreen" fontWeight="bold" exampleText="{ | }"/>
```
Comment (richardbuckle) [inline SpeechResponder/Cottle.xshd]: I'm not sure about this one.

File: SpeechResponder/Cottle.xshd [hunk 2]
```diff
@@ -38,6 +38,12 @@
       <Begin color="Delimiter">{</Begin>
       <End color="Delimiter">}</End>
     </Span>
+
+    <!-- delimiters not matched by any other rule are unexpected (i.e. a syntax error) -->
+    <Rule color="UnexpectedDelimiter">
```
Comment (richardbuckle) [inline SpeechResponder/Cottle.xshd]: Looks good.

File: SpeechResponder/Cottle.xshd (hunk 1 above)
Comment (Tkael) [reply SpeechResponder/Cottle.xshd]: Whoops, that was from testing last night. Forgot to revert that change.

Comment (Tkael) [reply SpeechResponder/Cottle.xshd]: Fix pushed.

**Pull request merged**